app.js

```diff
@@ -1 +1,52 @@
-console.log("js file is connected")
\ No newline at end of file
+checkedRadio = document.querySelector("input[name='ingredient']:checked");
+searchIngredient(checkedRadio);
+
+/**
+ * This function adds and ingredient a url, that is used to 
+ * make the call to the API for birthday cakes with that ingredient
+ * 
+ * @param {object} radioInput radio input DOM Element
+ */
+function searchIngredient(radioInput) {
+  const rootUrl = "https://api.edamam.com/search";
+  const recipeUrlElement = document.querySelector("#recipe-url");
+  const labelElement = document.querySelector("#label");
+  const imageElement = document.querySelector("#cake-img");
+  const ingredientsListElement = document.querySelector(".ingredients");
+
+  ingredientsListElement.innerHTML = "";
+  const apiUrl = `${rootUrl}?q=birthday+cake+${radioInput.value}&app_id=589ecbd6&app_key=6d6116bfcbdc60fe641222727dc9eb8f`;
+  fetch(apiUrl)
+    .then((response) => response.json())
+    .then((data) => {
+      console.log("Success:", data);
+      // The default number of search is 10 items, i.e when "from=" is specified but "to=" is not,
+      // or when neither "to=" or "from=" are specified in the apiUrl.
+      // With a free plan we are allowed to request max of 100 items.
+      // console.log(data.hits.length);
+
+      recipeUrlElement.setAttribute("href", data.hits[0].recipe.url);
+      labelElement.innerHTML = data.hits[0].recipe.label;
+      imageElement.src = data.hits[0].recipe.image;
+      let ingredientList = data.hits[0].recipe.ingredientLines;
+
+      // Since different cakes will have different amount of ingredients
+      // then this codes create list items according to the number of ingredients
+
+      ingredientList.forEach((elem) => {
+        const item = document.createElement("li");
+        ingredientsListElement.appendChild(item);
+        const lowCaseElem = elem.toLowerCase();
+        if (
+          lowCaseElem.includes("for the cake") ||
+          lowCaseElem.includes("for the frosting")
+        ) {
+          item.className = "heading-item";
```

Comment:
It's a good practice not to set the class attribute directly. Better:
```javascript
item.classList.add("heading-item");
```
The reason is that there could be other classes there we don't want to overwrite.